app/components-react/root/StartStreamingButton.tsx

```diff
@@ -114,6 +114,14 @@ export default function StartStreamingButton(p: { disabled?: boolean }) {
 
     if (!primaryPlatform) return false;
 
+    if (
+      !!UserService.state.auth?.platforms &&
+      StreamingService.views.isMultiplatformMode &&
+      Object.keys(UserService.state.auth?.platforms).length > 1
+    ) {
+      return true;
+    }
```

Comment:
Seems like these changes aren't meant to be part of this PR?

Reply:
Oh weird, I think I forgot to branch off of master since the fix is in https://github.com/stream-labs/desktop/commit/a3adc49cf966b00db74c4dd959e72f007706e94b already, which has been merged. I can remove this commit from the history?

Reply:
I think you can just merge master onto this branch and it should clear it up

Reply:
Ok, thanks @gettinToasty!

Reply:
Looks like this still isn't merged up to master.  LMK if you need help getting rid of this.